lib/files/copy.go

```diff
@@ -27,7 +36,7 @@ func Copy(ctx context.Context, dst io.Writer, src io.Reader, opts ...CopyOption)
 		// we allocate a buffer to use as a temporary buffer, rather than alloc new every time.
 		c.buffer = make([]byte, defaultBufferSize)
 	}
-	l := int64(len(c.buffer))
+	buflen := int64(len(c.buffer))
```

Comment:
From a different code review, I discovered the readily available ambiguity of the variable name `l` (U+006C) which is way to similar to `1` (U+0031) and `I` (U+0049), in the Github fonts.

So, I try to avoid single-letter variable names for l and I.